docker/dev/Dockerfile

```diff
@@ -26,7 +26,7 @@ COPY Makefile vpp.env ./
 
 RUN cp -r plugins/vpp/binapi /tmp/orig_binapi \
  && make generate-binapi \
- && diff -r plugins/vpp/binapi /tmp/orig_binapi
+ && diff -r -N -q plugins/vpp/binapi /tmp/orig_binapi
```

Comment:
This does not help with what you probably intended. The `patch` command used in Makefile target `verify-binapi` needs `--no-backup-if-mismatch` flag to not produce `*.orig` files on mismatch. Then the verification should succeed.